README.md

```diff
@@ -16,15 +16,15 @@
 
 ## Building components in Node.js
 
-If you plan to build a component for [elastic.io platform](http://www.elastic.io) in Node.js then you can visit our dedicated documentation page which describes [how to build a component in node.js](https://support.elastic.io/support/solutions/articles/14000027123-how-to-build-a-component-in-node-js).
+If you plan to build a component for [elastic.io platform](http://www.elastic.io) in Node.js then you can visit our dedicated documentation page which describes [how to build a component in node.js](https://docs.elastic.io/guides/building-nodejs-component.html).
 
```

Comment:
The link must be without the `HTML` (https://docs.elastic.io/guides/building-nodejs-component)